### PR TITLE
feat(seo): add ga support

### LINKED ISSUE #20: SEO: Add Google analytics support
This is a
[ ] bug
[x] feature request
[ ] documentation change
[ ] build/deploy enhancement request

Description:
Add support for Google Analytics. As part of this story add a test API key hardcoded. Later we will pick up the API key from a config file in CI/CD.
Add meta tags to index.html to add project description and keywords

### PR DESCRIPTION
**Pull request by priyankajayaswal**
Fixes #20 
- Added GA support
- Updated README.md to give instructions on how to use GA and Auth0 with references

### CODE REVIEW

File: .env.local.sample [hunk 1]
```diff
@@ -0,0 +1,6 @@
+AUTH0_SECRET='use [openssl rand -hex 32] to generate a 32 bytes value'
+AUTH0_BASE_URL='http://localhost:3000'
+AUTH0_ISSUER_BASE_URL='https://YOUR_DOMAIN'
+AUTH0_CLIENT_ID='YOUR_CLIENT_ID'
+AUTH0_CLIENT_SECRET='YOUR_CLIENT_SECRET'
```
Comment (souvikbasu) [inline .env.local.sample:5]: are these lines needed as part of this issue? I think they should already be there from Auth0 integration PR

File: README.md [hunk 2]
```diff
@@ -50,6 +50,24 @@ npm install
 npm run dev
 ```
 
+### To use Auth0
+
+- Use  `wallora.test@gmail.com` to login to auth0 [website](https://auth0.auth0.com/u/login/) using google. 
+- Copy `.env.local.sample` to `.env.local` and populate the fields from the app you see in auth0 post login.
+- After this when you run the dev server and try to login you should be able to see the auth0 login flow.
+
+Reference: https://auth0.com/docs/quickstart/webapp/nextjs
```
Comment (souvikbasu) [inline README.md:59]: this should have ideally been added as part of Auth0 integration. Will add it in this PR

**Review by souvikbasu — CHANGES_REQUESTED** [review]

File: .env.local.sample (hunk 1 above)
Comment (priyankajayaswal) [reply .env.local.sample:5]: yes. I see that this file was removed from master . We need to have this to keep track of all config variables we eventually use in our app. In Auth0 I had not added the readme hence added it here.

File: README.md (hunk 2 above)
Comment (priyankajayaswal) [reply README.md:59]: as per feedback was asked to add it. Since this isn't any feature, bug etc. Adding it with this PR itself.

**Review by souvikbasu — APPROVED** [review]

**Pull request merged**